foundations/the_basics/how_this_course_will_work.md

```diff
@@ -20,6 +20,12 @@ This curriculum works by pulling together the best content from across the inter
 
 Most lessons will contain questions that you should be able to answer before moving on.  Some of the lessons will ask you to complete exercises. In addition, we provide several projects throughout the curriculum to help you grow your understanding by actually building things.
 
+### A Note About Language
+
+The Odin Project attracts people from all over the world who aspire to learn how to program. Please be aware that this curriculum is written in English and maintained by people whose primary (and typically only) language is English. If you are a non-English speaker, or English is not a primary language for you, we *highly* recommend spending extra time on topics and terms that you do not understand right away. We also encourage you to seek out additional resources that teach about these topics in your own native language so you can understand them more fully.
```

Comment:
It's imperative to point out somewhere here why English is important to programming at-large.

(e.g. official documentation/references/and the community at-large [Odin and beyond] speak or read it).

Other than that, seems good.